bindata/network/openshift-sdn-azure/azure-drop-icmp-daemonset-sdn.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: add-iptables
```

Comment:
It's weird to write this out as its own separate object when the script is so dependent on the specifics of the script in the DaemonSet. I feel like it would be better to just have the DS `cat` this out to a file at startup...